include/diagnostics/geweke.hpp

```diff
@@ -0,0 +1,64 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2020 Vissarion Fisikopoulos
+// Copyright (c) 2018-2020 Apostolos Chalkis
+
+//Contributed and/or modified by Alexandros Manochis, as part of Google Summer of Code 2020 program.
+
+//Licensed under GNU LGPL.3, see LICENCE file
+
+
+#ifndef GEWEKE_HPP
+#define GEWEKE_HPP
+
+#include <boost/math/distributions/fisher_f.hpp>
+
+template <typename VT, typename MT, typename NT>
+bool perform_geweke(MT const& samples, NT const& frac1, NT const& frac2)
```

Comment:
please provide some reference of the method (as you did for the R interface) and a comment on what is `frac1`, `frac2`.
Should `frac1`, `frac2` have default values? Also is there any other better name, e.g. `frac_first`, `frac_last` ?

Reply:
What is this function doing exactly? What is the meaning of the returned bool ?

Reply:
I have added comments to provide information about geweke diagnostic